Libraries/Text/TextCodeBlock.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict-local
+ */
+
+import type {ColorValue} from 'react-native/Libraries/StyleSheet/StyleSheet';
+
+export type TextCodeBlockProp = $ReadOnly<{|
+  /**
+   * The background color of the text code block.
+   */
+  backgroundColor?: ?ColorValue,
+
+  /**
+   * The border color of the text code block.
+   */
+  borderColor?: ?ColorValue,
+
+  /**
+   * The border radius of the text code block.
+   */
+  borderRadius?: ?number,
+|}>;
```

Comment:
If we are talking about flow types only I don't see anything wrong with defining props here rather than extending it. Backing this by facebook/rn codebase: https://github.com/facebook/react-native/blob/f16348ca033bb29a5c750faf7b8e99eaef8fdfed/Libraries/Components/StatusBar/StatusBar.js#L63

Reply:
In order to make `textCodeBlock` boolean:
- I have to disable background color at the View level (Text) altering the API
- Implement conversion at TextAttribute leve

Approach I'm taking here is to extend the functionality rather than modify it.

Reply:
I see @azimgd point, not sure we want to change the API, but, the check on 
```js
/*
   * When `true`, draws inline border around text mostly used for inline code blocks.
   */
   textCodeBlock?: ?TextCodeBlockProp,
   ```

Maybe needs some clarification, the comments does not seem to be correct, as we are no checking for it to be `true`?

In all honesty this code goes a bit over my head 😅

Reply:
Comment is fixed